src/task/task.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "https://raw.githubusercontent.com/microsoft/azure-pipelines-task-lib/master/tasks.schema.json",
   "id": "907d3b28-6b37-4ac7-ac75-9631ee53e512",
   "name": "PRMetrics",
-  "friendlyName": "PR Metrics v1.5.9",
+  "friendlyName": "PR Metrics v1.5.10",
```

Comment:
Please revert this change and the other version update. I'll be updating the project in January via an automated process, so I'd prefer to hold off for a full update until then

Reply:
FYI: I had updated this just from the contribution guide saying we should 🙂

https://github.com/microsoft/PR-Metrics/blob/main/.github/CONTRIBUTING.md#updating-an-extension